src/popup/stores/AccountDetailStore.ts

```diff
@@ -39,6 +43,14 @@ export default class AccountDetailStore {
     chrome.runtime.sendMessage({ type: MESSAGE_TYPE.GET_MORE_TXS });
   }
 
+  public removeTokenAtIndex = (index: number) => {
+    this.tokens.splice(index, 1);
```

Comment:
keep all the biz logic in the background scripts. reason for this is so that we don't have to do splicing in two places. 

lets do all the heavy lifting in the bg script and have the bg script datastore be the `single source of truth`. so doing the remove token from the UI should follow this pattern:
1. ui -> sendMessage -> remove token from bg
2. bg removes token from its datastore
3. bg -> sendMessage -> with new token list
4. ui -> handleMessage -> replace `this.tokens` with new list